lecture/1-prologue/1-prologue.tex

```diff
@@ -171,20 +171,36 @@
 
 \begin{frame}{When? - Dates}
 	% Dates
-	\begin{itemize}
-		\item \textbf{Lecture} - Start in Calendar Week 17 (Today)
-		      \begin{itemize}
-			      \item Monday, 10:15 - 11:45 (Online via Zoom) \\
-			            {\color{gray}Lecturers: M. B. Sigl and D. Probst}
-		      \end{itemize}
-		\item \textbf{Exercises} - Start in Calendar Week 18
-		      \begin{itemize}
-			      \item Wednesday, 14:15 - 15:45 (H8) \\
-			            {\color{gray}Tutor: D. Probst}
-			      \item Friday, 16:15 - 17:45 (Online via Zoom) \\
-			            {\color{gray}Tutors: M. B. Sigl and D. Probst}
-		      \end{itemize}
-	\end{itemize}
+	\begin{columns}
+		\begin{column}{0.5\textwidth}
+			\begin{itemize}
+				\item \textbf{Lecture} - Start in Calendar Week 16 (Today)
+				      \begin{itemize}
+					      \item Friday, 10:15 - 11:45 (H8) \\
+					            {\color{gray}Lecturers: M. B. Sigl and D. Probst}
+				      \end{itemize}
+				\item \textbf{Exercises} - Start in Calendar Week 17
```

Comment:
I would prefer if the exercises were all on one slide (i.e. probably next) slide so that as a student you have an immediate overview of all the exercises.